packages/xo-server-usage-report/report.html.tpl

```diff
@@ -575,17 +575,17 @@
             <tr>
               <th>UUID</th>
               <th>Name</th>
-              <th>Total space (GiB)</th>
-              <th>Used space (GiB)</th>
-              <th>Free space (GiB)</th>
+              <th>Total space</th>
```

Comment:
I still wonder if it's good to have heterogeneous prefixes in a columns…

Reply:
@olivierlambert Would do you think about this? Should we make this change?

Reply:
It's too much out of context to me to have an opinion. Feel free to add these kind of "product" oriented questions in a list so that I can have a quick face to face debrief on these :+1: (and make decisions based with a context). Thanks!

Reply:
The objective of this PR is to use `human-format` for all stats. Here is a sample of the result. 

![image](https://user-images.githubusercontent.com/21563339/46467264-9a8daf80-c7cd-11e8-9115-38250e1ec8dd.png)

As you see in the picture, it will have on the same column values with different prefixes.